src/index.html

```diff
@@ -2,21 +2,107 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"/>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"
     />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/ua/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="form">
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple_1/"
+              class="nav__link nav__link--active'"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/brands/samsung/"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/telecommunication/smart/"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/comp-and-periphery/"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/gadgets/"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/tablet_el_knigi/tablet/"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.moyo.ua/ua/foto_video/photo_video/"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="video">
```

Comment:
Use semantic class names that represent the meaning of the content. The class name 'video' for the <li> element could be more descriptive of its role within the navigation.